presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -133,6 +133,7 @@
     private boolean createsOfNonManagedTablesEnabled = true;
 
     private boolean tableStatisticsEnabled = true;
+    private int partitionStatisticsSampleSize = 100;
```

Comment:
why default is not a `Integer.MAX_VALUE`?

Reply:
Because that would be a bad default. I would say we're fixing a "bug" here, we want this to be on by default.